### DISCUSSION TITLE
Anywhere I could find the E3V2-BLTouch-5x5-v4.2.2 configuration files?

### DISCUSSION THREAD

**Discussion by FirmwareScrub** [Jyers/Marlin#1461] [Q&A] [answered]
First time trying to alter the firmware on my printer, and currently in over my head. I am currently running the 5x5 4.2.2 precompiled firmware but I wanted to update just a couple settings on it (preheat settings order and temp sensor change mainly). I initially tried to adjust one of the templates included, but them I realized I have no way of double checking it. On top of this I'm not sure the settings are correct (probing speed for example is the same in the BLT 3x3 standard and high speed variants if I'm not mistaken). 

Hoping the config file is available somewhere.

**Comment by Cal-LF** [comment, marked as answer]
The E3V2 templates are in the configuration folder. 3X3 is what is offered. It can be changed to 5X5 in the config.h file at 1593 #define GRID_MAX_POINTS  -  pick 5 to change it from 3.

**Comment by gilegue** [comment]
> First time trying to alter the firmware on my printer, and currently in over my head. I am currently running the 5x5 4.2.2 precompiled firmware but I wanted to update just a couple settings on it (preheat settings order and temp sensor change mainly). I initially tried to adjust one of the templates included, but them I realized I have no way of double checking it. On top of this I'm not sure the settings are correct (probing speed for example is the same in the BLT 3x3 standard and high speed variants if I'm not mistaken).
> 
> Hoping the config file is available somewhere.

What do you mean by temp sensor change? Do you refer to temp sensor type or PID change? As for high speed variants, all you have to do is define it in Configuration_adv.h line# 861 (uncomment #define BLTOUCH_HS_MODE) to use it. But the different is just not store the probe each time. Errr unless you are referring to probe travel speed which is another kind of thing altogether.

**Reply by FirmwareScrub** [reply]
> What do you mean by temp sensor change? Do you refer to temp sensor type or PID change? As for high speed variants, all you have to do is define it in Configuration_adv.h line# 861 (uncomment #define BLTOUCH_HS_MODE) to use it. But the different is just not store the probe each time. Errr unless you are referring to probe travel speed which is another kind of thing altogether.

Ah, ok, you may have resolved my paranoia about the settings not being correct. I thought high speed mode had to do with  X and Y travel speeds between probes in configuration.h line# 1176.  The temp sensor change I referred to is just updating configuration.h line#486 temp_sensor_0 from the stock thermister to a new one I've acquired. 

Thanks guys. I think I'll give the files another pass and try again.
